cedar-policy/src/api.rs

```diff
@@ -1206,6 +1206,23 @@ impl SchemaFragment {
         })
     }
 
+    #[cfg(feature = "deprecated_unspecified")]
+    #[deprecated]
+    /// Create an `SchemaFragment` from a JSON value (which should be an
+    /// object of the shape required for Cedar schemas).
```

Comment:
```suggestion
    /// Create a `SchemaFragment` from a JSON value (which should be an
    /// object of the shape required for Cedar schemas). This function is provided
    /// to provide backwards-compatible behavior with Cedar version 3.x. It will
    /// be removed in a future release.
```